include/vast/Dialect/LowLevel/LowLevelOps.td

```diff
@@ -257,4 +260,45 @@ def InlineScope
     let regions = (region AnyRegion:$body);
 }
 
+def LowLevel_FuncOp : FuncOp< LowLevel_Dialect, "func", [] >
+{
+  let skipDefaultBuilders = 1;
+
+  let builders = [OpBuilder< (ins
```

Comment:
Does function builder differ anyhow between `HL_FUncOp` `LL_FuncOp` and `HLABI_FuncOp` if not thay can be moved to base?

Reply:
`ABIFuncBase` does use different builder from the other two.